docs/migrating.md

```diff
@@ -53,7 +53,7 @@ The output provides a description of the necessary changes or removals required
 [REMOVED] button--primary -> use Warp button component instead
 [REMOVED] button--small -> use Warp button component instead
 ```
-Look in the [CSS docs](https://warp-ds.github.io/css-docs/) to find the correct class names, and in [Tech docs](https://warp-ds.github.io/tech-docs/) for the component names. 
+Look in the [CSS docs](https://warp-ds.github.io/css-docs/) to find the correct class names, and in these [Tech docs](https://warp-ds.github.io/tech-docs) for the components.
```

Comment:
should it be "... and in `the` [Tech docs]( ..." ?

Reply:
I was thinking to emphasise that it's the tech docs we're in but I see now that "these" doesn't really suggest it 😄 I'll leave "the" then and not overthink this.

Reply:
[a454263](https://github.com/warp-ds/tech-docs/pull/47/commits/a454263ecefbd6f2c383a3d40e381e3007487067)